publish.go

```diff
@@ -0,0 +1,135 @@
+// This package is used to patch release of private repositories, pulling the assets that
+// should be used by brew into this repo. It is used for `qlik-cli`.
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+)
+
+const path = "./bin/qlik-cli/"
+
+var token string
+
+type Release struct {
+	AssetsURL string `json:"assets_url"`
+	Tag       string `json:"tag_name"`
+}
+
+type Asset struct {
+	URL  string `json:"url"`
+	Name string `json"name"`
+	Tag  string
+}
+
+func main() {
+	token = os.Getenv("GITHUB_TOKEN")
+	if token == "" {
+		check(fmt.Errorf("GITHUB_TOKEN not set"))
+	}
+	base := "https://api.github.com/repos/qlik-trial/qlik-cli"
```

Comment:
can we consume the base url to the private repo from an env variable instead?

Reply:
Sure thing